Makefile

```diff
@@ -5,23 +5,31 @@ UID = $(shell id -u)
 BUILD_FOLDER = $(PWD)/build/$(NAMESPACE)
 ROOT_DIR = $(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
 
+BUILD_TYPE := $(or ${BUILD_TYPE},standalone)
 TARGET := $(or ${TARGET},minikube)
 PROFILE := $(or $(PROFILE),minikube)
 KUBECTL=kubectl -n $(NAMESPACE)
 
+ifeq ($(BUILD_TYPE), standalone)
+    UNIT_TEST_TARGET = unit-test
+else
+    UNIT_TEST_TARGET = convert-coverage
+endif
+
 ifeq ($(TARGET), minikube)
 ifdef E2E_TESTS_MODE
 E2E_TESTS_CONFIG = --img-expr-time=5m --img-expr-interval=5m
 endif
 define get_service
 minikube -p $(PROFILE) service --url $(1) -n $(NAMESPACE) | sed 's/http:\/\///g'
 endef # get_service
-VERIFY_CLUSTER = _verify_cluster
+VERIFY_CLUSTER = _verify_minikube
```

Comment:
VERIFY_MINIKUBE ?
Where is this required?

Reply:
It was verify_minikube a week ago,
@YuviGold  changed it to VERIFY_CLUSTER but if we want to eliminate passing the kubeconfig from the host to the skipper container (Which is mandatory for MAC since the port is different) we have to update the kubeconfig file by the command 'minikube update-context' to point to the minikube cluster)

Reply:
@YuviGold can you please change the BUILD_TYPE of the CI? 
We can't pass the unit-tests for this PR until then since it doesn't create the coverage file